portfolio/src/test/java/FiltersTest.java

```diff
@@ -0,0 +1,49 @@
+import static org.junit.Assert.assertEquals;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.sps.data.SearchQuery;
+import java.util.Arrays;
+import java.util.List;
+import java.net.MalformedURLException;
+import java.util.stream.Collectors;
+import org.junit.Assert;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+public class FiltersTest {
+    @Test
+    public void FilterTest1() throws MalformedURLException{
+        String title = "Taylor Swift - Blank Space";
```

Comment:
In general, it's better to use clearly mocked titles and descriptions for unit tests.